### ISSUE TITLE
Inefficient proquest

### ISSUE THREAD

**Issue by RubenBarrionuevo** [JCulley/ProQuests-Documentation#204]
Spigot Username: https://www.spigotmc.org/members/pichurrin.316310/
Spigot Version: 1.12.2
Plugin Version: 2.0.7.1

I create this problem again due to the null support I received from the author of this plugin.
Of 4 reports that I have published in this repository, only 1 has been solved and in a costly and very heavy way, ultimately complicated.

Certain incidents that have not been resolved are the following:
- https://github.com/JCulley/ProQuests-Documentation/issues/187
- https://github.com/JCulley/ProQuests-Documentation/issues/185
- https://github.com/JCulley/ProQuests-Documentation/issues/115

And only one, I have been solved and as you can see I had to cry blood to achieve it.
- https://github.com/JCulley/ProQuests-Documentation/issues/183

However, I am still waiting for the author to finish a wiki in conditions one day, because I see many empty sections.
In addition to seeking help through this repository and to limit the author to tell you "I have already labeled the issue, now that someone else helps you"

Sections without content:
- https://github.com/JCulley/ProQuests-Documentation/wiki/In-Game-Creator
- https://github.com/JCulley/ProQuests-Documentation/wiki/Actions
- https://github.com/JCulley/ProQuests-Documentation/wiki/Team-Types Teams? and this how it works, are we fortune tellers?
- https://github.com/JCulley/ProQuests-Documentation/wiki/Actions if I want to see the current progress of the mission? Do I have to limit myself to the events that I have here? Or can you not give help to use JS variables?

But well, I do not want to focus on my problems either, here I put a list of problems of other users that due to their clumsiness or lack of practice their report has been closed.
- https://github.com/JCulley/ProQuests-Documentation/issues/193 (This user is absolutely right about his work.)
- https://github.com/JCulley/ProQuests-Documentation/issues/184 (It has a problem, but since it does not comply with the template I close the issue and solved it.)
- https://github.com/JCulley/ProQuests-Documentation/issues/180 (This also happened to me, is it still failing? laughing ...)

In short, if I keep looking for sure I find more users with problems without solving, these are the ones I found on the first page.

It is worth highlighting:
- Your plugin is not simple
- Your wiki is incomplete
- The assistance you give and other possible users is null
- It's the plugin that I regret having bought
- You are a person with a complicated interest with which to communicate with affected users

I thought about giving this done, but I will not until I obtained a plugin that meets certain minimum requirements that must comply.
See you.

**Comment by R0Nii69** [comment]
Hello, I am currently part of the Staff of a large server, which has invested money in this plugin in which, to our surprise, it now turns out not to provide support when errors arise,
I wanted to know if there is any way to solve or respond to our issues, since we have paid for something that does not work.

The staff team of the server has already published 4 reports on which only had  1 (1/4).


Plug-in version: 2.0.7.1



Certain incidents that have not been resolved are the following:

# 187
# 185
# 115

And only one, I have been solved and as you can see I had to cry blood to achieve it.

# 183
However, I am still waiting for the author to finish a wiki in conditions some day, because I see many empty sections.
In addition to seeking help through this repository and limiting the author to say "I already labeled the problem, now that someone else helps."

Sections without content:

https://github.com/JCulley/ProQuests-Documentation/wiki/In-Game-Creator
https://github.com/JCulley/ProQuests-Documentation/wiki/Actions
https://github.com/JCulley/ProQuests-Documentation/wiki/Team-Types Teams? And that's how it works, are we fortune tellers?
https://github.com/JCulley/ProQuests-Documentation/wiki/Actions if I want to see the current progress of the mission? Do I have to limit myself to the events I have here? Or can not you give help to use the JS variables?

Other users also have the same problems as us, here are some examples:

# 193 (This user is absolutely right about his work.)
# 184 (It has a problem, but since it does not comply with the template, I close the problem and solve it.)
# 180 (This also happened to me, is it still failing? Laughing ...)
In summary, if I keep looking for sure that I find more users with unsolved problems, these are the ones I found on the first page.

No matter what else we have to do or publish to get solutions and answers to our posts, but if they do not do something I will have to contact the other affected people and take some measure, reimbursement, advertising so that nobody else buys the plugin or something.

Greetings I hope you read this post and you answer us.

**Comment by JCulley** [comment]
Closing this as it’s not a suggestion or an error.
